.stylelintrc.json

```diff
@@ -1,21 +1,22 @@
 {
```

Comment:
- [Optional] Please in all future projects, write a meaningful title for your pull request. Your PR title should be a short description of changes introduced in the pull request. This will make your repo more professional. Also, kindly make your PR description to be more descriptive. The description should be a summary of all the changes implemented in the pull request. This will make your repo more professional. Kindly check [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more information about professional repo.

- [Optional] Please in all future projects, kindly make sure that your branch name is the name of a feature or action you are adding to the branch. This is best practice for [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md#feature-branches). A good branch name can be setup-mobile-portfolio.

- [Optional] Please always use descriptive commit messages - each commit should be small enough to describe it in one sentence. The commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). This will make your code more professional. A good example of a descriptive commit message is `Add markup for index.html` and a bad example is `added file`.